### DISCUSSION TITLE
negate filter

### DISCUSSION THREAD

**Discussion by bfd69** [gotson/komga#1523] [Ideas]
### Describe your suggested feature

hellooo, is it possible : 
1: to have a negate filter ? the purpose it to have all books that don't match the filter 
2: to be possible to use filter along side with selection, actually if i use a filter and then select books, the filter button disappears and inversely

i'm using the version 1.9.2
thanks a lot for reading me !

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by Blazeflack** [comment]
I would love to be able to use "negative" filters on my tags for example. In an ideal world it would be possible to easily mix and match "positive" and "negative" filters.

Some other solutions solve this by allowing 1 click for a positive filtering, and 2 clicks for a negative filtering. A 3rd click clears the filtering for that specific item.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.15.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.15.0))
